sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/UIUtils.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.streaming.ui
+
+import org.apache.spark.sql.streaming.StreamingQuery
+
+private[ui] object UIUtils {
```

Comment:
In this PR, we moved many classes from `streaming.ui` to `spark.ui`.
Instead of making this new class, I'd like to recommend to put the below two functions to `spark.ui.UIUtils`.

Reply:
IMHO, there is no need to move these two functions to `spark.ui.UIUtils`, as there are only used in `sql` package. Besides if we moved them to `spark.ui.UIUtils`, there will be a dependency cycle, as `withNoProgress` used `StreamingQuery` class